utils/auth/strategies/basic.js

```diff
@@ -0,0 +1,30 @@
+const passport = require('passport');
+const { BasicStrategy } = require('passport-http');
+const boom = require('@hapi/boom');
+const bcrypt = require('bcrypt');
+
+const UsersService = require('../../../components/users/usersService');
+
+passport.use(
+  new BasicStrategy(async function (email, password, cb) {
+    const userService = new UsersService();
+
+    try {
+      const user = await userService.getUser({ email });
+
+      if (!user) {
+        return cb(boom.unauthorized(), false);
```

Comment:
add feedback

Reply:
I dont understand because if you not authorize you receive messages